packages/foreman/rubygem-rubyforge/rubygem-rubyforge.spec

```diff
@@ -1,100 +1,107 @@
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name rubyforge
 
-# Depency loop here. Kill test when resolving dependency loop
-# is needed
-%global enable_test 0
-
-Summary:       A script which automates a limited set of rubyforge operations
-Name:          %{?scl_prefix}rubygem-%{gem_name}
-Version:       2.0.4
-Release:       8%{?dist}
-Group:         Development/Languages
-License:       MIT
-URL:           http://rubyforge.org/projects/codeforpeople
-Source0:       http://gems.rubyforge.org/gems/%{gem_name}-%{version}.gem
-BuildRoot:     %{_tmppath}/%{pkg_name}-%{version}-%{release}-root-%(%{__id_u} -n)
-Requires:      %{?scl_prefix_ruby}ruby(release)
-Requires:      %{?scl_prefix_ruby}rubygems
-Requires:      %{?scl_prefix_ruby}rubygem(json) >= 1.1.7
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-%if %{enable_test}
-BuildRequires(check): rubygem(rake)
-BuildRequires(check): rubygem(json)
-# The following line causes dependency loop
-BuildRequires(check): rubygem(hoe)
-%endif
-BuildArch:     noarch
-Provides:      %{?scl_prefix}rubygem(%{gem_name}) = %{version}
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.0.4
+Release: 9%{?dist}
+Summary: A script which automates a limited set of rubyforge operations
+Group: Development/Languages
+License: MIT
+URL: http://codeforpeople.rubyforge.org/rubyforge/
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby 
+Requires: %{?scl_prefix_ruby}ruby(rubygems) 
+Requires: %{?scl_prefix}rubygem(json) >= 1.1.7
```

Comment:
@zjhuntin this seems to be wrong. I'd say it was supposed to be `scl_prefix_ruby`.

Reply:
It is in the prefixes:
https://github.com/theforeman/foreman-packaging/blob/1868ff28e10ca612c5101a073621e20cc98e7b70/scl_prefixes.json#L3
Has this been done manually rather than calling `./add_gem_package.sh`?